test/e2e/framework/skipper/skipper.go

```diff
@@ -136,28 +127,15 @@ func SkipUnlessAtLeast(value int, minValue int, message string) {
 	}
 }
 
-// SkipUnlessLocalEphemeralStorageEnabled skips if the LocalStorageCapacityIsolation is not enabled.
-func SkipUnlessLocalEphemeralStorageEnabled() {
-	if !utilfeature.DefaultFeatureGate.Enabled(localStorageCapacityIsolation) {
-		skipInternalf(1, "Only supported when %v feature is enabled", localStorageCapacityIsolation)
-	}
-}
-
-func SkipUnlessDownwardAPIHugePagesEnabled() {
-	if !utilfeature.DefaultFeatureGate.Enabled(downwardAPIHugePages) {
-		skipInternalf(1, "Only supported when %v feature is enabled", downwardAPIHugePages)
-	}
-}
-
-func SkipUnlessExecProbeTimeoutEnabled() {
-	if !utilfeature.DefaultFeatureGate.Enabled(execProbeTimeout) {
-		skipInternalf(1, "Only supported when %v feature is enabled", execProbeTimeout)
+func SkipUnlessFeatureGate(gate featuregate.Feature) {
+	if !utilfeature.DefaultFeatureGate.Enabled(gate) {
+		skipInternalf(1, "Only supported when %v feature is enabled", gate)
 	}
 }
 
-func SkipIfCSIMigrationEnabled() {
-	if utilfeature.DefaultFeatureGate.Enabled(csiMigration) {
-		skipInternalf(1, "Only supported when %v feature is disabled", csiMigration)
+func SkipIfFeatureGate(gate featuregate.Feature) {
```

Comment:
Same here, please add comment.

Reply:
The comment of course explains it. I'm thinking of the code lines where the function is called.